setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import find_packages,setup
 
 setup(name='arm_diags',
-      version='2.0',
+      version='3.0',
```

Comment:
version='4.0'?

Reply:
Yes, it should be version='4.0'. Thank you for your review, Jill!